src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_RENT = 40;
+  const LONG_TEMR = 7;
```

Comment:
There is a typo in the constant name `LONG_TEMR`. It should be `LONG_TERM` to correctly represent 'long term'.

Reply:
Fix it